### DISCUSSION TITLE
Question python

### DISCUSSION THREAD

**Discussion by ProtonDev123** [mezotv/chalk-advanced#1] [Q&A] [answered]
Does this also work in python

**Comment by mezotv** [comment]
No it does not work in python its node.js only!

**Reply by ProtonDev123** [reply]
thanks

**Comment by finnkoetting** [comment, marked as answer]
It is only working with node.js and probably with yarn. Its not that hard to code it yourselve I think 😉

**Reply by ProtonDev123** [reply]
thanks man!
